java/ql/test/query-tests/Telemetry/AutomodelFrameworkModeExtraction/com/github/codeql/test/NonPublicClass.java

```diff
@@ -0,0 +1,10 @@
+package com.github.codeql.test;
+
+/**
+ * No candidates in this class, as it's not public!
+ */
+class NonPublicClass {
+  public void noCandidates(String here) {
+    System.out.println(here);
```

Comment:
making this class would lead to the `String here` parameter being selected as a candidate. Why do you think this means the example code should be tweaked?

Reply:
Making this class `public` would lead to the `String here` parameter being selected as a candidate. Why do you think this means the example code should be tweaked?

Reply:
Further implementation detail: the reason is that the class is not considered externally visible by the `ModelApi` class that `@jcogs33` has built. We use this class to not select candidates that won't be callable from the outside.